perllib/FixMyStreet/Cobrand/FixMyStreet.pm

```diff
@@ -59,5 +59,26 @@ sub get_country_for_ip_address {
     return mySociety::Gaze::get_country_from_ip($ip);
 }
 
+sub report_form_extras {
+    ( { name => 'gender', required => 0 }, { name => 'variant', required => 0 } )
+}
+
+sub ask_gender_question {
+    my $self = shift;
+
+    return 1 unless $self->{c}->user;
+
+    my $reports = $self->{c}->model('DB::Problem')->search({
+        user_id => $self->{c}->user->id,
+        extra => { like => '%gender%' }
+    });
```

Comment:
It'd be marginally faster, for users with lots of reports, to use `order_by => { -desc => 'id' }` here.